oep_framework/oep/BNBCameraOrientation.h

```diff
@@ -0,0 +1,12 @@
+// AUTOGENERATED FILE - DO NOT MODIFY!
```

Comment:
Why do you need this file here if it is already present in BanubaEffectPlayer.framework